core-blocks/gallery/edit.js

```diff
@@ -264,7 +264,7 @@ class GalleryEdit extends Component {
 						</li>
 					) ) }
 					{ isSelected &&
-						<li className="blocks-gallery-item">
+						<li className="blocks-gallery-item has-add-item-button">
```

Comment:
I don't understand this class name exactly... wouldn't `is-selected` or something like that be better? Or I guess this is saying it's surrounding the `add-item-button`, but this `has-` prefix makes this read to me like it could have or could not have that button.

What about `blocks-gallery-add-item-button`?

Reply:
I wish we could lint these somehow, I always forget them as my brain is still adjusting from my old class name habits!

Thanks for catching that. 👍

Reply:
It's doable. I think @aduth has implemented something similar for Calypso a long time ago. This rule is following [BEM]((http://getbem.com/introduction/)) (Blocks, Elements, Modifiers), which is pretty well established standard.

Reply:
Hi @tofumatt thanks for the review. I used `has-add-item-button` instead of `is-selected` because it Is a different existing behaviour than the selected ítem. Also, yes. A gallery ítem like that could or not have that button (this same element type Is used to contain the images). The button per se was already 100% width, but its container wasn't. Does this make sense for you guys? @gziolo